packages/flutter/test/material/scaffold_test.dart

```diff
@@ -501,11 +501,11 @@ void main() {
 
     final SemanticsTester semantics = new SemanticsTester(tester);
     await tester.pumpWidget(new MaterialApp(home: new Scaffold(
-      body: new Semantics(label: bodyLabel, child: new Container()),
-      persistentFooterButtons: <Widget>[new Semantics(label: persistentFooterButtonLabel, child: new Container())],
-      bottomNavigationBar: new Semantics(label: bottomNavigationBarLabel, child: new Container()),
-      floatingActionButton: new Semantics(label: floatingActionButtonLabel, child: new Container()),
-      drawer: new Drawer(child:new Semantics(label: drawerLabel, child: new Container())),
+      body: const Text(bodyLabel),
+      persistentFooterButtons: <Widget>[const Text(persistentFooterButtonLabel)],
+      bottomNavigationBar: const Text(bottomNavigationBarLabel),
+      floatingActionButton: const Text(floatingActionButtonLabel),
+      drawer: const Drawer(child:const Text(drawerLabel)),
```

Comment:
this is fine, but why this change?

Reply:
Previously, the `Semantics` widgets were creating `SemanticsNode`s with a size of `Rect.zero`. With the new definition of visibility introduced in this PRs, those `SemanticsNode`s are invisible and are no longer allowed in the tree (ensured with asserts). Rather than giving the `Container` in the old code a width and a height, I felt like the code would be easier to read if I instead replace the entire thing with a `Text` widget that has an intrinsic width/height.